src/libmongoc/tests/mock_server/mock-server.c

```diff
@@ -868,6 +868,7 @@ request_assert_no_duplicate_keys (request_t *request)
 }
 
 
+/* Note that mock-rs.c has another, similar function of the same name: */
```

Comment:
This comment is misleading, please remove; the function in `mock-rs.c` is named `mock_rs_receives_request`, as are other mock replica-set counterparts to mock server functions (`mock_server_*` vs. `mock_rs_*`).

Reply:
Ping: not yet addressed.